modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/SearchTransactionRequest.scala

```diff
@@ -0,0 +1,14 @@
+package org.tessellation.rosetta.server.model.search
+
+/** This is an intermediary object used to hold the values from the Rosetta request that is relevant to our system.
+  */
+
+case class SearchTransactionRequest(
+  operation: SearchTransactionTransactionRequestOperator,
```

Comment:
so it's an operation or an operator? field name specify it differently that the type